cookbook/how_to_upgrade_to_new_version.rst

```diff
@@ -0,0 +1,224 @@
+.. index::
+    single: Upgrade
+
+How to Upgrade to a New Version
+===============================
+
+1. Checkout from the GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve a new version and upgrade your OroCRM instance, please execute the following steps:
+
+**1** Create backups of your Database and Code.
+
+**2** "Cd" to the crm root folder and switch the application to the maintenance mode.
+
+.. code-block:: bash
+
+    $ cd /path/to/application
+    $ sudo -uwww-data app/console lexik:maintenance:lock --env prod
+
+**3** Stop the cron tasks
+
+.. code-block:: bash
+
+    $ crontab -e -uwww-data
+
+    .. note::
+        www-data can be changed to user under which your web server runs
+
+Comment this line.
+
+.. code-block:: text
+
+     */1 * * * * /usr/bin/php /path/to/application/app/console --env=prod oro:cron >> /dev/null
+
+Kill the related job daemon process.
+
+.. code-block:: bash
+
+    $ ps ax|grep php5
+    $ kill <process_pid>
+
+<process_pid> - is a PID of currently executing application job daemon process. For example:
+
+.. code-block:: text
+
+    /path/to/application/app/console jms-job-queue:run --max-runtime=3600 --max-concurrent-jobs=5 --env=prod
+
+**4** Pull changes from the repository.
+
+.. code-block:: bash
+
+    $ sudo -uwww-data git pull
+    $ sudo -uwww-data git checkout <VERSION TO UPGRADE>
+
+**5** Upgrade the composer dependency and set correct owner to the retrieved files.
```

Comment:
Upgrade the composer dependency and set up the right owner to the retrieved files.